pkg/printers/internalversion/printers_test.go

```diff
@@ -26,6 +26,8 @@ import (
 
 	"github.com/google/go-cmp/cmp"
 
+	"k8s.io/utils/ptr"
+
```

Comment:
Probably my fault.  Maybe just do a `hack/update-gofmt.sh` run?

Reply:
damn, so easy to skim over imports when reviewing

Reply:
fixed